include/redisserver.h

```diff
@@ -596,6 +614,12 @@ class RedisServer {
         inline static const std::string _MODEL_TIMEOUT_ENV_VAR =
             "SR_MODEL_TIMEOUT";
 
+        /*!
+        *   \brief Environment variable for thread count in thread pool
+        */
+        inline static const std::string _TP_THREAD_COUNT =
+            "SR_THREAD_COUNT";
```

Comment:
Out of scope for this PR, but at some point we should document what environment variables the client initialization depends on. I vaguely wonder if we should create specific methods to initialize these internal variables from the environment or some other method so that we can be flexible on how the clients can be configured.

Reply:
Agreed, let's make sure to catch this variable when we do the variables documentation prior to the release